spec/maestro_spec.rb

```diff
@@ -0,0 +1,51 @@
+ENV["RACK_ENV"] = "test"
+
+require "rspec"
+require "rack/test"
+require "json"
+require "pry-byebug"
+require_relative "../app"
```

Comment:
I guess this is one of the downfalls of Sinatra, no?

Reply:
Somewhat. I don't see a huge problem with it, but I might refactor it at some point later. Rails does have a nice way of handling this, and I'm sure we could set it up similarly here.